core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -764,4 +773,78 @@ class UserServiceSpec extends AnyFlatSpecLike with TestDriverComponent with Mock
       actual.errorReport.statusCode.get shouldEqual StatusCodes.NotFound
     }
   }
+
+  behavior of "listBillingProjectsV2"
```

Comment:
We had no unit level coverage of the existing method, this is adding some.

Reply:
Consider testing the addition of the azure billing project (when enabled).

Reply:
Done in https://github.com/broadinstitute/rawls/pull/1690/commits/02e527dbdae779c9eab3db7b4c0724b6d7254828